irodsConnector/session.py

```diff
@@ -1,16 +1,19 @@
 """ session operations
 """
-import logging
+import warnings
+import os
+import json
 import irods.session
 from irods.exception import NetworkException
-from keywords import exceptions
+from .keywords import exceptions
 
 class Session:
     """Irods session authentication.
 
     """
 
-    def __init__(self, irods_env: dict, irods_env_path='', password='') -> irods.session:
+    def __init__(self, irods_env: dict=None, irods_env_path: str='', 
```

Comment:
Mypy will not like this. If you have the choice between `dict` and `None`, it expects `Optional[dict]` as a type hint. Where `Optional` is imported from the `typing` standard library.